prefs.js

```diff
@@ -126,6 +126,10 @@ var TabsContainer = GObject.registerClass(
             this.stackSwitcher = new Gtk.StackSwitcher({
                 margin_left: 16,
                 margin_right: 16,
+                margin_top: 8,
+                margin_bottom: 8,
+                halign: Gtk.Align.CENTER,
+                valign: Gtk.Align.CENTER,
```

Comment:
I'm not sure we want it floating in the center

Reply:
That's how this widget usually is aligned in Gnome apps, but on a HeaderBar, so I thought the same thing was attempted here. Like in Gnome Software, System Monitor, Photos, Videos, etc.